generators/app/package-json.js

```diff
@@ -37,7 +37,8 @@ const DEFAULTS = {
     'videojs-generate-karma-config',
     'not-prerelease',
     'sinon',
-    'videojs-standard'
+    'videojs-standard',
+    'npm-merge-driver'
```

Comment:
super small module that makes rebasing projects that use `package-lock` super easy. Basically it will merge package-locks when they conflict much faster than regenerating them ourselves.